msteams-platform/bots/bring-bot-to-teams.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Bring your bot app to Teams
+description: Learn about bringing your existing bot app to the Teams environment
+ms.localizationpriority: high
+ms.date: 09/25/2024
+ms.topic: conceptual
+ms.author: surbhigupta
+---
+
+# Bring your bot app to Teams
+
+[**TBA** - ]
```

Comment:
So early reviewers better understand the content IA with this TOC reorg, could you jot down a quick summary of what this article should cover? Thanks!

Reply:
@erikadoyle Updated as suggested